client/src/components/ProgressBar/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Progress } from 'antd';
+import BackLink from '../BackLink';
+import { Container, Content } from './style';
+import { Body16B } from '../Typography';
+
+const ProgressBar = ({ currentLocation, stepes, percent }) => {
+  return (
+    <Container stepes={stepes}>
```

Comment:
```suggestion
    <Container steps={steps}>
```

and check if used in other places

Reply:
done